spec/workers/modify_project_updated_at_worker_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+
+describe ModifyProjectUpdatedAtWorker do
+    let(:worker) { described_class.new }
+    let(:project) { create :project }
+
+    describe 'project touch timestamp' do
+        before do
+          allow(Project).to receive(:find).and_return(project)
```

Comment:
Not blocking: i'd just inline this to the spec as there is only one, i'd extract to the before block for reuse if / when needed.